src/aux-server/aux-web/aux-player/interaction/DragOperation/PlayerFileDragOperation.ts

```diff
@@ -226,18 +232,35 @@ export class PlayerFileDragOperation extends BaseFileDragOperation {
     protected _onDragReleased(calc: FileCalculationContext): void {
         super._onDragReleased(calc);
 
+        this._files[0].tags[this._context + '.x'] = this._toCoord.x;
+        this._files[0].tags[this._context + '.y'] = this._toCoord.y;
+
+        const fileTemp = createFile(this._files[0].id, this._files[0].tags);
```

Comment:
Can this be rewritten to not edit the `tags` on `this._files[0]`?

Maybe try something like this:
```javascript
const fileTemp = createFile(this._files[0].id, {
  ...this._files[0].tags,
  [this._context + '.x']: this._toCoord.x,
  [this._context + '.y']: this._toCoord.y,
});
```

Reply:
I'll change that and see how it goes